src/index.js

```diff
@@ -0,0 +1,71 @@
+/** Copyright (c) 2017 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const requiredLabels = [
+  'breaking',
+  'feature',
+  'bugfix',
+  'docs',
+  'discussion',
+  'release',
+  'prerelease',
+  'greenkeeping',
+];
+
+module.exports = robot => {
+  robot.on('pull_request.opened', check);
+  robot.on('pull_request.reopened', check);
+  robot.on('pull_request.edited', check);
+  robot.on('pull_request.synchronize', check);
+  robot.on('pull_request.unlabeled', check);
+  robot.on('pull_request.labeled', check);
+
+  async function check(context) {
+    const pr = context.payload.pull_request;
+
+    // set status to pending while checks happen
+    setStatus(context, {
+      state: 'pending',
+      description:
+        'Checking whether at least one required semver-related label exists',
+    });
+
+    function getRequiredLabels(pr) {
+      const labels = pr.labels;
+      const filteredLabels = labels.filter(l =>
+        requiredLabels.includes(l.name),
+      );
+      return filteredLabels;
+    }
+
+    const hasRequiredLabel = getRequiredLabels(pr).length > 0;
+    if (hasRequiredLabel) {
+      // set status to success
+      return setStatus(context, {
+        state: 'success',
+        description: 'At least one required semver-related label exists',
+      });
+    } else {
+      // failure - missing label
+      return setStatus(context, {
```

Comment:
Hm, thoughts on having the probot add a comment with a list of approved labels in the failure case?

Reply:
That might be nice